src/components/ComposerQuill/composer.js

```diff
@@ -66,9 +84,12 @@ class Composer extends React.Component {
     const {send} = this.props;
 
     return () => {
-      const text = this.quill.root.textContent.trim();
+      const text = this.quill.root.textContent;
```

Comment:
why don't we need `.trim()` anymore?

Reply:
There was nothing to trim, so it wasn't doing anything

Reply:
does it hurt to leave the `trim()` there? seems like maybe there is a scenario where we have trailing whitespace or something?